packages/shopping/src/controllers/user.controller.ts

```diff
@@ -260,4 +261,46 @@ export class UserController {
 
     return {token};
   }
+
+  @put('/users/password-reset', {
+    responses: {
+      '200': {
+        description: 'The updated user profile',
+        content: {
+          'application/json': {
+            schema: UserProfileSchema,
+          },
+        },
+      },
+    },
+  })
+  async passwordReset(
```

Comment:
To make sure the user is updating it's own password, you can inject the CURRENT_USER like [this](https://github.com/strongloop/loopback4-example-shopping/blob/cf7678baaa5dcf64c7ff93728f32f545064cb6c2/packages/shopping/src/controllers/user.controller.ts#L142), then compare the user email.

Reply:
Noted will update accordingly